src/desktop/apps/galleries_institutions/queries/partner_categories_query.coffee

```diff
@@ -1,14 +1,14 @@
 module.exports =
 
   """
-  query GalleriesInstitutionsPartnerCategoriesQuery($category_type: CategoryType, $type: [PartnerClassification]){
-    partner_categories(category_type: $category_type, size: 50, internal: false){
+  query GalleriesInstitutionsPartnerCategoriesQuery($categoryType: PartnerCategoryType, $type: [PartnerClassification]){
+    partner_categories: partnerCategories(categoryType: $categoryType, size: 50, internal: false){
       name
-      id
-      primary: partners(eligible_for_listing: true, eligible_for_primary_bucket: true, type: $type, sort: RANDOM_SCORE_DESC, default_profile_public: true) {
+      id: slug
```

Comment:
This change is a little weird-- we use this to populate our list of partners/categories on the server, then the `partner_filter_query` below to populate on the client, and we need a way to connect the two.

With v2, we no longer allow you to query for an ID as part of aggregations (you just have `value`, which matches the `slug` here).